README.md

```diff
@@ -34,6 +34,8 @@ pcre2             has pcre2 regex engine
 ...
 ```
 
+You may need to build Universal Ctags from source. In that case, to enable `pcre2`, run the configure script like so: `./configure --enable-pcre2`.
```

Comment:
I followed the instruction from <https://docs.ctags.io/en/latest/autotools.html>, and I didn't need to specify `--enable-pcre2`. It seems that it's enabled automatically if libpcre2 is installed. Can you check if this is necessary for you? 

Also, can you try installing with Homebrew? The documentation recommends it. <https://docs.ctags.io/en/latest/osx.html#building-with-homebrew>.

Reply:
I installed it with homebrew per the instructions, but that builds _without_ pcre2 support. You may be right that specifying `--enable-pcre2` is not required, but I think it will cause the configure stage to fail if `pcre2` is not present.